samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/EventLogListener.java

```diff
@@ -145,14 +160,14 @@ private void processPageLoad(EventLogBean eventLog) {
 	 * @param assessmentTitles List of titles
 	 * @return
 	 */
-	private List<SelectItem> initAssessmentListFilter(List<Object[]> assessmentTitles) {
+	private List<SelectItem> initAssessmentListFilter(List<Object[]> assessmentTitles, Map<Long,Integer> statusMap) {
 	   List<SelectItem> assessments = new ArrayList<SelectItem>();
 	   
 	   String allStr = ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EventLogMessages", "filterAll");
       assessments.add(new SelectItem((long)-1, allStr));
       for (Object[] assessment : assessmentTitles) {
          Long id = (Long)assessment[0];
-         String title = (String)assessment[1];
+         String title = (String)assessment[1]+((statusMap.get(id)!=null && statusMap.get(id)==2)?"-deleted":"");
```

Comment:
You're performing the `.get( id )` on the `HashMap<>` twice here; this is inefficient. Instead, extract the return value and check the value:

```
Integer status = statusMap.get( id );
String strStatus = status != null && status == 2 ? "-deleted" : "";
String title = (String) assessment[1] + strStatus;
```